synopsys-task/src/synopsys-task/synopsys-bridge.ts

```diff
@@ -73,6 +73,35 @@ export class SynopsysBridge {
     }
 
     try {
+      if (inputs.ENABLE_NETWORK_AIR_GAP) {
+        if (inputs.SYNOPSYS_BRIDGE_PATH.length !== 0) {
+          executable = await this.setBridgeExecutablePath(
```

Comment:
can we validate SYNOPSYS_BRIDGE_PATH as well, and log the error 
and if executable not found, throw error relatable to that